api/v1alpha1/etcdpeer_types.go

```diff
@@ -7,10 +7,26 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// BootstrapPeer describes a single *other* peer that will be used for bootstrapping
+type BootstrapPeer struct {
+	// Name is the initial name of the peer, which is set into etcd at the
+	// end of this process
```

Comment:
Not sure what this means - `Name is a friendly name for the peer, used as a means to identify the peer once it has joined a cluster`